app/controllers/wizard/steps/measure_amount_controller.rb

```diff
@@ -31,7 +31,7 @@ def applicable_measure_units
       def measure_amount_answers
         return {} unless params.key?(:wizard_steps_measure_amount)
 
-        params.require(:wizard_steps_measure_amount).permit(*applicable_measure_unit_keys)
+        params.require(:wizard_steps_measure_amount).permit(*applicable_measure_unit_keys).to_h
```

Comment:
Why is `.to_h` needed here?

Reply:
I wanted something I could compare with a hash in the tests. Happy to revert. It's functionally exactly the same from the perspective of the model